dynawo/sources/Common/DYNTrace.h

```diff
@@ -318,12 +318,29 @@ class Trace {
   /**
    * @brief test if a log exists
    *
+   * This tests only the file logs
+   *
    * @param tag : Tag added to the log, can be used as a filter in logging sinks.
    * @param slv : Severity level.
    * @return true if this log with this level exists
    */
   static bool logExists(const std::string& tag, SeverityLevel slv);
 
+  /**
+   * @brief Test if a standard log exists
+   *
+   * This test the level of the standard output log
+   *
+   * @param slv Severity level
+   * @returns whether the standard log accepts the log level
+   */
+  static bool standardLogExists(SeverityLevel slv) {
+    return slv >= defaultLevel_;
```

Comment:
Won't we see debug logs dumped in the standard output when info is given in the .jobs file for example?

Reply:
The default log level is compiled-time and depends on _DEBUG_ macro.
If a debug log is requested, standardLogExists will return false so it only depends on the job configuration (log files). If only info logs are defined in the .jobs file, the whole condition (see TraceStream constructor) will be false and the log will be filtered. If a debug log file is allowed with this tag, the log will be duplicated on standard output